<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>malikDialogBase</class>
 <widget class="QDialog" name="malikDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>618</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>malik_plugin</string>
  </property>
  <widget class="QPushButton" name="start_btn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>570</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab">
   <property name="geometry">
    <rect>
     <x>-70</x>
     <y>-160</y>
     <width>441</width>
     <height>311</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>441</width>
     <height>311</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>441</width>
     <height>311</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>1.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="show_btn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>590</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Audio &amp; Video</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hide_btn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>570</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Only Audio</string>
   </property>
  </widget>
  <widget class="QVideoWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>29</y>
     <width>711</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pause_btn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>570</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
  <widget class="QPushButton" name="play_btn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>590</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stop_btn">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>570</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="open_btn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>570</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
  </widget>
  <zorder>lab</zorder>
  <zorder>start_btn</zorder>
  <zorder>show_btn</zorder>
  <zorder>hide_btn</zorder>
  <zorder>widget</zorder>
  <zorder>pause_btn</zorder>
  <zorder>play_btn</zorder>
  <zorder>stop_btn</zorder>
  <zorder>open_btn</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>PyQt5.QtMultimediaWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
